eng/scripts/dependencies/compare-deps-files.ps1

```diff
@@ -0,0 +1,53 @@
+param(
+  [string]$PSDepsFile,
+  [string]$AzSdkDepsFile
+)
+
+function Get-DepsFileContent($depsfile)
+{
+  if ($depsfile.StartsWith("http")) {
+    try {
+      $response = Invoke-WebRequest -Uri $depsfile
```

Comment:
Add `-UseBasicParsing` or lack of legacy IE could throw an exception (probably will happen sooner or later).

Reply:
From what I read this parameter is now deprecated in PS 6+ https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-webrequest?view=powershell-7.

Reply:
Old habits, I guess. I saw it recently, but maybe I was using **Windows** PowerShell for some reason (hardly ever do that anymore).